cmd/jb/install_test.go

```diff
@@ -35,10 +36,9 @@ func TestInstallCommand(t *testing.T) {
 		ExpectedJsonnetLockFile []byte
 	}{
 		{
-			Name:                    "NoURLs",
-			ExpectedCode:            0,
-			ExpectedJsonnetFile:     []byte(`{"dependencies":null}`),
-			ExpectedJsonnetLockFile: []byte(`{"dependencies":null}`),
```

Comment:
Why don't we expect a lockfile anymore? Shouldn't this yield a empty lockfile as well?

Reply:
I understand that the [current intention of the install command](https://github.com/jsonnet-bundler/jsonnet-bundler/blob/4de4096004cc1c403a13358356d32598063bdc77/cmd/jb/install.go#L40-L43) was to work even if the lockfile is missing. As we ignored the error of [Load()](https://github.com/jsonnet-bundler/jsonnet-bundler/blob/4de4096004cc1c403a13358356d32598063bdc77/pkg/jsonnetfile/jsonnetfile.go#L38-L41), this would initialize an empty lockfile.
Before this change, this empty lockfile was written unconditionally by the installCommand. Now that we write only if there is a diff, there is no diff between empty value lockfile and default value, so no lockfile is written. 

If we were to warrant an empty lockfile (alongside the empty jsonnetfile), I think we should add it in the `initCommand`.

Reply:
I‘m fine with this 😃